### PR TITLE
472 update readmemd

### LINKED ISSUE #472: update README.md
- [x] 理解 README 相關資訊（整理出較好的寫法）
- [x] Create a branch for this issue
- [x] 編輯 README.md ：說明開啟專案的步驟
- [x] PR

### PR DESCRIPTION
**Pull request by godmmt**
### DEVELOP
- [x] 472 update readmemd

### CHECK LIST
- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component: 0
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### UML

- [Class Diagram link](https://lucid.app/lucidchart/7f239953-18e5-41c9-94ed-648e3b3bb53d/edit?viewport_loc=-1651%2C-537%2C1923%2C1267%2C0_0&invitationId=inv_040636e5-7726-4c0c-8b2b-5230594910e8)
- [Sequence Diagram link](https://lucid.app/lucidchart/c918bc56-b89a-4d6f-8493-5037df815983/edit?viewport_loc=2172%2C1298%2C1732%2C1093%2C0_0&invitationId=inv_2ef49e7c-5f81-45de-b53e-667b661043fe)

### CODE REVIEW

**Comment by godmmt** [comment]
# [我的需求] 需要 Vercel 團隊權限

截圖 2024-01-12 下午5 42 23

我的 GitHub Email: 
```
mingreal75@gmail.com
```

File: src/services/chatgpt_service.ts [hunk 1]
```diff
@@ -1,9 +1,12 @@
 // chatGPTService.ts
 import axios from 'axios';
+import { ILiteMessage } from '../interfaces/message';
 
-const getBotResponse = async (userInput: string) => {
+const getBotResponse = async (messages: ILiteMessage[]) => {
   const apiUrl = '/api/v1/messages';
-  const requestData = { userInput };
+  const requestData = { messages };
+  // eslint-disable-next-line no-console
```
Comment (Luphia) [inline src/services/chatgpt_service.ts]: - remove console.log

File: src/pages/index.tsx [hunk 2]
```diff
@@ -44,22 +44,29 @@ const Homepage = () => {
     const pendingMessage: IMessage = {
       id: Math.random(),
       content: '...',
-      sender: 'lizzz',
+      role: 'assistant',
```
Comment (Luphia) [inline src/pages/index.tsx]: - declare static parameter

File: src/pages/index.tsx [hunk 3]
```diff
@@ -44,22 +44,29 @@ const Homepage = () => {
...
-    const response: IMessage = await getBotResponse(userInput);
+    const response: IMessage = await getBotResponse(liteMessages);
 
     // InFo: step 5 - Get response from backend (20240102 - Liz)
 
     const responseMessage: IMessage = {
       id: response.id,
       content: response.content,
-      sender: 'lizzz',
+      role: 'assistant',
```
Comment (Luphia) [inline src/pages/index.tsx]: - declare static parameter

**Review by Luphia — CHANGES_REQUESTED** [review]

**Pull request merged**